build/scripts/ci-release-image.sh

```diff
@@ -11,21 +11,21 @@ QUAY_BOT_USER=${QUAY_BOT_USER:-kubectltrace_buildbot}
 git_ref=$1             # github.ref        format: refs/REMOTE/REF
                        #                       eg, refs/heads/BRANCH
                        #                           refs/tags/v0.9.6-pre
-git_base_ref=$(basename ${git_ref})
+git_base_ref=$(basename "${git_ref}")
 
 make=$(command -v make)
 docker=$(command -v docker)
 
 makeopts=""
-if [[ ! -z "${git_base_ref}" ]]; then
+if [[ -n "${git_base_ref}" ]]; then
   makeopts="-e GIT_BRANCH=${git_base_ref} ${makeopts}"
 fi
 
-if [[ ! -z "$QUAY_TOKEN" ]]; then
+if [[ -n "$QUAY_TOKEN" ]]; then
   $docker login -u="${GIT_ORG}+${QUAY_BOT_USER}" -p="$QUAY_TOKEN" quay.io
-  $make $makeopts image/push
+  $make "$makeopts" image/push
```

Comment:
I think this is what is causing CI failure
```suggestion
  $make $makeopts image/push
```